checkout/views.py

```diff
@@ -0,0 +1,190 @@
+from django.shortcuts import render, redirect, reverse
+from django.conf import settings
+from django.contrib import messages
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+
+from .forms import OrderForm
+from .models import Order, OrderLineItem
+from products.models import Product
+from profiles.models import UserProfile
+from profiles.forms import UserProfileForm
+
+
+def checkout(request):
+
+	if request.method == 'POST':
+		cart = request.session.get('cart', {})
+
+		form_data = {
+			'full_name': request.POST['full_name'],
+			'email': request.POST['email'],
+			'phone_number': request.POST['phone_number'],
+			'country': request.POST['country'],
+			'postcode': request.POST['postcode'],
+			'town_or_city': request.POST['town_or_city'],
+			'street_address1': request.POST['street_address1'],
+			'street_address2': request.POST['street_address2'],
+			'county': request.POST['county'],
+		}
+
+		order_form = OrderForm(form_data)
+		if order_form.is_valid():
+			order = order_form.save()
+			for item_id, item_data in cart.items():
+				try:
```

Comment:
I'd suggest replacing moving the `except` directly up here and unindenting the rest

Reply:
Good idea.